Add notes to docs around installing the gem (#708)
DataDog/puppet-datadog-agent: README.md

To enable reporting of Puppet runs to your Datadog timeline, enable the report processor on your Puppet master and reporting for your clients. The clients send a run report after each check-in back to the master.

1. Install the [dogapi][3] gem on your system. You'll need to restart puppetserver after the gem has been installed for it to be loaded. 

> If you're configuring the dogapi gem via code, you could do this via a notify:

Reviewer: ```suggestion
If you're configuring the dogapi gem by code, you can do this with notify:
```

    Class Datadog_reports is already defined in Puppet::Reports
    ```

> If you don't see any reports coming in, check your puppet server logs.

Reviewer: ```suggestion
If you don't see any reports coming in, check your Puppet server logs.
```
